### PR TITLE
docs: update question details screenshot

### PR DESCRIPTION
**Pull request by groupsvkg**
first PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ This project aims to develop a question-and-answer web app where users can solve
 
 ### Home
 
+**Updated Question details ui**
```
Comment (prabhatg) [inline README.md:40]: Change the heading to add more detail

File: README.md [hunk 2]
```diff
@@ -37,6 +37,10 @@ This project aims to develop a question-and-answer web app where users can solve
...
+
+<img src='./design/ui/home_iteration_13.png' width='400'>
```
Comment (prabhatg) [inline README.md:42]: Why width is 400?

File: design/ui/home_iteration_13.png
Comment (prabhatg) [inline design/ui/home_iteration_13.png:1]: What is Part-2?

**Review by prabhatg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (groupsvkg) [reply README.md:40]: Update it.

File: README.md (hunk 2 above)
Comment (groupsvkg) [reply README.md:42]: It because all other images are of same width and occupies smaller area

File: design/ui/home_iteration_13.png
Comment (groupsvkg) [reply design/ui/home_iteration_13.png:1]: I am working on it .. will be part of next PR.

**Review by prabhatg — APPROVED** [review]
Thank you for your work

**Pull request merged**